src/components/templates/Auditorium/Auditorium.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+import { SectionPreviews } from "./components/SectionPreviews";
+import { Section } from "./components/Section";
+
+import { AuditoriumVenue } from "types/venues";
+
+const enum AuditoriumVideoPosition {
+  TOP = "top",
+  CENTER = "center",
+}
+
+export interface AuditoriumProps {
+  venue: AuditoriumVenue;
+}
+
+/*
+  TODO: Sections will be stored as a collection of a Auditorium venue.
+  Each should have its own autogenerated id.
+  Each will contain chats.
+
+  I guess we should have more stricter view choice. For example: 6, 10, 20, 30+ sections.
+  This way we can provide best UI for each(most used) case
+*/
+
+export const Auditorium: React.FC<AuditoriumProps> = ({ venue }) => {
+  // self-explainatory name. I wonder if I should have this both for FULL and SECTION view?
+  const { videoPosition, sections } = venue;
+
+  const [chosenSectionId, setChosenSectionId] = useState<string | undefined>();
```

Comment:
What is the type of `sections` here? Are they already the full details? If so, would it make sense to pass that full object down to the `Section` component, rather than just the id?

Reply:
It was removed.